likecoin/js/admin-settings/src/pages/LikecoinButtonPage.js

```diff
@@ -0,0 +1,214 @@
+import {
+  useState, useEffect, useMemo,
+} from 'react';
+import axios from 'axios';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { debounce } from 'lodash';
+import Section from '../components/Section';
+import LikecoinInfoTable from '../components/LikecoinInfoTable';
+import SubmitButton from '../components/SubmitButton';
+import LikeButtonPreview from '../components/LikeButtonPreview';
+import SettingNotice from '../components/SettingNotice';
+import LikecoinHeading from '../components/LikecoinHeading';
+import { SITE_LIKER_INFO_STORE_NAME } from '../store/site-likerInfo-store';
+import { USER_LIKER_INFO_STORE_NAME } from '../store/user-likerInfo-store';
+
+function LikecoinButtonPage() {
+  const {
+    DBSiteLikerId,
+    DBSiteLikerAvatar,
+    DBSiteLikerDisplayName,
+    DBSiteLikerWallet,
+    DBSiteLikerIdEnabled,
+  } = useSelect((select) => select(SITE_LIKER_INFO_STORE_NAME).selectSiteLikerInfo());
+
+  const {
+    DBUserLikerId,
+    DBUserLikerAvatar,
+    DBUserLikerDisplayName,
+    DBUserLikerWallet,
+  } = useSelect((select) => select(USER_LIKER_INFO_STORE_NAME).selectUserLikerInfo());
+
+  const { postUserLikerInfo } = useDispatch(USER_LIKER_INFO_STORE_NAME);
+
+  const [siteLikerIdEnabled, enableSiteLikerId] = useState(DBSiteLikerIdEnabled);
+  // If siteLikerId is enabled === not editable,
+  // then overwrite the user liker info with site liker info
+  const defaultLikerId = siteLikerIdEnabled
+    ? DBSiteLikerId
+    : DBUserLikerId;
+  const defaultLikerDisplayName = siteLikerIdEnabled
+    ? DBSiteLikerDisplayName
+    : DBUserLikerDisplayName;
+  const defaultLikerWalletAddress = siteLikerIdEnabled
+    ? DBSiteLikerWallet
+    : DBUserLikerWallet;
+  const defaultLikerAvatar = siteLikerIdEnabled
+    ? DBSiteLikerAvatar
+    : DBUserLikerAvatar;
+  const [likerIdValue, getLikerIdValue] = useState(defaultLikerId);
```

Comment:
Naming `get` is confusing for the function that update the state
```suggestion
  const [likerIdValue, setLikerIdValue] = useState(defaultLikerId);
```